lib/devbox_launcher/models/box.rb

```diff
@@ -38,6 +38,7 @@ def start_cmd
       args = {
         project: config[:project],
         account: account,
+        zone: (config[:zone] || 'us-central1-a'),
```

Comment:
I don't think we should default since the wrong one will break the command and not allow gcloud to try its own default
```suggestion
        zone: config[:zone],
```